docs/ref/contrib/postgres/indexes.txt

```diff
@@ -24,3 +24,28 @@ available from the ``django.contrib.postgres.indexes`` module.
     PostgreSQL. You can install it using the
     :class:`~django.contrib.postgres.operations.BtreeGinExtension` migration
     operation.
+
+``BRINIndex``
+=============
+
+.. class:: BRINIndex(pages_per_range=None)
+
+..  versionadded:: 1.11
+
+    Creates a `BRIN index
+    <https://www.postgresql.org/docs/devel/static/brin-intro.html>`_. It is not
+    an optimal index to use unless your table is very large.  The `page inspect
+    module <https://www.postgresql.org/docs/devel/static/pageinspect.html>`_
+    contains useful functions for debugging and inspection of the indexes and
+    tables. This module also contains useful functions for
+    :ref:`django.contrib.postgres.indexes.GinIndex` introspection. Because they
+    are only meant for system administration tasks, the functionality of the
+    module is not implemented.
+
+    To get the full advantage of using ``BRIN`` it is recommended to tweak the
+    index construction and consider the nature of the tables before using this
+    index. ``BRIN`` indexes take an optional input parameter ``pages_per_range``
+    for. The `Python sweetness`_ blog has a post with further details on the
+    ``pages_per_range`` parameter.
+
+    .. _Python sweetness: http://pythonsweetness.tumblr.com/post/119568339102/block-range-brin-indexes-in-postgresql-95
```

Comment:
OK. My remark in the comment was intended as a question whether this was allowed.